README.md

```diff
@@ -1,18 +1,18 @@
 <div align="center">
   <h3>Part of:</h3>
-  <img width="150"src="https://github.com/ahmfaiz/wire-scraper/blob/main/wire-logo-rounded-small.png" alt="wire-logo" />
+  <img width="150" src="https://github.com/ahmfaiz/wire-scraper/blob/main/wire-logo-rounded-small.png" alt="wire-logo" />
   <h1>The <a href="https://github.com/ahmfaiz?tab=repositories&q=wire&type=&language=&sort=">WIRE</a> Project</h1>
   <p>
     Web Info Retrieval Engine
   </p>
   <a href="https://opensource.org/">
     <img src="https://badges.frapsoft.com/os/v3/open-source.svg?v=103" alt="Open Source Love" />
   </a>
-  <a href="https://opensource.org/license/gpl-3-0/">
-    <img src="https://img.shields.io/badge/license-GPLv3-blue" alt="Licence" />
+  <a href="https://opensource.org/licenses/gpl-3.0/">
+    <img src="https://img.shields.io/badge/license-GPLv3-blue" alt="License" />
   </a>
 </div>
 
-![](https://i.imgur.com/waxVImv.png)
+<img src="https://i.imgur.com/waxVImv.png" alt="Web Dashboard" />
```

Comment:
As this is a Markdown file, markdown code is preferred. Above HTML code was used for alignment reasons.
Kindly edit your pr.